comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -291,14 +294,26 @@ func (rc rcClient) agentConfigUpdateCallback(updates map[string]state.RawConfig,
 
 		// Need to update the log level even if the level stays the same because we need to update the source
 		// Might be possible to add a check in deeper functions to avoid unnecessary work
-		err = rc.settingsComponent.SetRuntimeSetting("log_level", mergedConfig.LogLevel, model.SourceRC)
+		if err := rc.settingsComponent.SetRuntimeSetting("log_level", mergedConfig.LogLevel, model.SourceRC); err != nil {
+			errs = append(errs, err)
+		}
+	}
+
+	streamLogsSource := config.Datadog().GetSource("logs_config.streaming.enable_streamlogs")
+	if mergedConfig.EnableStreamLogs {
+		if err := rc.settingsComponent.SetRuntimeSetting("enable_streamlogs", mergedConfig.EnableStreamLogs, model.SourceRC); err != nil {
```

Comment:
I think you also need to register the setting here: https://github.com/datadog/datadog-agent/blob/d2e2032c39896c491805aba5e2fdfa7bf0070488/cmd/agent/subcommands/run/command.go#L456-L469